pkg/runtime/wfengine/component.go

```diff
@@ -130,6 +130,18 @@ func (c *workflowEngineComponent) Terminate(ctx context.Context, req *workflows.
 	return nil
 }
 
+func (c *workflowEngineComponent) Purge(ctx context.Context, req *workflows.PurgeRequest) error {
+	if req.InstanceID == "" {
+		return fmt.Errorf("a workflow instance ID is required")
```

Comment:
```suggestion
		return errors.New("a workflow instance ID is required")
```

If no format is included, please use `errors.New` (it's faster and makes less allocs)